### ISSUE TITLE
Custom Stickers & Emoticons (MSC2545) like it works in Cinny or FluffyChat

### ISSUE THREAD

**Issue by foresle** [vector-im/element-android#8060]
#### What would you like to do?

I would like to see native stickers and custom emoticons support in Element, like it works in Cinny or FluffyChat.

![зображення](https://user-images.githubusercontent.com/123166857/216193241-a7b331f6-b621-42e1-88f2-d2bafca32f9a.png)

#### Why would you like to do it?

Clients like Cinny already using native sticker packs instead of integrations in Element. It's work faster and have simplified manage process. 

#### How would you like to achieve it?

Cinny using [this specs](https://github.com/Sorunome/matrix-doc/blob/soru/emotes/proposals/2545-emotes.md), and it's work perfectly. I also write a [small tool](https://github.com/foresle/matrix_stickers_manager) to manage sticker packs. Need to just copy logic to Element :)

I can't write code for this for itself that I don't know Kotlin, but think it's a very important feature, so I created this issue.

### Are you willing to provide a PR?

No

**Comment by GithubUser96** [comment]
I think this'd be great too especially knowing that an existing, functional, Matrix compatible implementation already exists.

**Comment by lukasfri** [comment]
Its a Duplicate to #6193

**Comment by Nostramadeus** [comment]
> I think this'd be great too especially knowing that an existing, functional, Matrix compatible implementation already exists.

100%!

**Comment by StrikerTheHedgefox** [comment]
I am just stopping by to say I support this. Especially now that Discord is getting ready to shoot itself in the foot with a 4-bore shotgun.

**Comment by Spacefreak18** [comment]
there's sort of a workaround for this, if you use synapse and mautrix-discord:

https://github.com/mautrix/discord/issues/82

**Comment by StrikerTheHedgefox** [comment]
Not really. That's only useful for people trying to bridge the two or puppeting a discord account with matrix.
